edb/server/defines.py

```diff
@@ -29,7 +29,8 @@
 EDGEDB_SUPERGROUP = 'edgedb_supergroup'
 EDGEDB_SUPERUSER = s_def.EDGEDB_SUPERUSER
 EDGEDB_TEMPLATE_DB = s_def.EDGEDB_TEMPLATE_DB
-EDGEDB_SUPERUSER_DB = 'edgedb'
+EDGEDB_OLD_DEFAULT_DB = 'edgedb'
+EDGEDB_SUPERUSER_DB = 'main'
```

Comment:
`EDGEDB_SUPERUSER_DB` is also used in 2 functions in `metaschema.py`, should we patch up those 2 functions?

Reply:
I'm not sure if those functions are even right, really?

Reply:
Hmmm... and I bet that patches don't work in single-db mode

Reply:
Probably true, we don't have a patch test CI on single-db mode. Though the regular single-db CI has been green.